app/helpers/manageiq/providers/redfish/button/physical_server_button.rb

```diff
@@ -0,0 +1,7 @@
+module ManageIQ::Providers::Redfish
+  class Button::PhysicalServerButton < ::ApplicationHelper::Button::ButtonWithoutRbacCheck
+    def visible?
+      ::ManageIQ::Providers::Redfish::PhysicalInfraManager::PhysicalServer.any?
```

Comment:
I was thinking about adding some RBAC filtering around this, but I am worried that this might end up as an expensive operation for a solution which is far from ideal. cc @lpichler @kbrock

Reply:
Should this be in ui-classic on the physical_server controller?  Seems like it should see if any physical servers support the feature the button is implementing?

Reply:
No, this button and the underlying feature is tied to Redfish only.

Reply:
the api doesn't really do a good job running rbac on sub collections. we need a better story than adding rbac directly to the classes